<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SyrahMail</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="WriteMail">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>写邮件</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>211</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>261</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>70</y>
      <width>441</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QWebView" name="webView">
       <property name="url">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>949</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSyrahMail">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuWhatever">
     <property name="title">
      <string>Open</string>
     </property>
     <addaction name="actionWhatever_2"/>
     <addaction name="actionWhatever_3"/>
    </widget>
    <addaction name="actionWrite"/>
    <addaction name="separator"/>
    <addaction name="menuWhatever"/>
    <addaction name="actionSetup"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAccounts">
    <property name="title">
     <string>Accounts</string>
    </property>
    <addaction name="actionAdd_mail_account"/>
   </widget>
   <addaction name="menuSyrahMail"/>
   <addaction name="menuAccounts"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionWrite">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Write</string>
   </property>
  </action>
  <action name="actionWhatever_2">
   <property name="text">
    <string>whatever</string>
   </property>
  </action>
  <action name="actionWhatever_3">
   <property name="text">
    <string>whatever</string>
   </property>
  </action>
  <action name="actionSetup">
   <property name="text">
    <string>Add account</string>
   </property>
  </action>
  <action name="actionAdd_new_account">
   <property name="text">
    <string>Add new account</string>
   </property>
  </action>
  <action name="actionAdd_mail_account">
   <property name="text">
    <string>Add mail account</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionWrite</sender>
   <signal>triggered()</signal>
   <receiver>WriteMail</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
